### PR TITLE
[PAGOPA-1564] feat: added update_cup_iban.py script to update cup iban in ecConfigTable

### PR DESCRIPTION
**Pull request by aferracci**
#### List of Changes

- Added `update_cup_iban.py` script to update the iban codes in `ecConfigTable` with the last choice of the CI (CUP label)

#### Motivation and Context

Update the CUP iban in the config table

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: scripts/update_cup_iban.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                        print(f"[INFO][update_entities] iban {entity['Iban']} is already updated with CUP iban {cup_iban}")
+
+        except Exception as e:
+            print(f"[ERROR][update_entities] HTTP error occurred: {e}")
+
+    def get_iban_by_ci(self, id_ci):
+
+        print(f"[INFO][get_iban_by_ci] getting CUP iban for {id_ci} EC")
+        try:
+            api_url = f"{self.apim_url}/creditorinstitutions/{id_ci}/ibans/enhanced?label=0201138TS"
```
Comment (pasqualespica) [inline scripts/update_cup_iban.py:55]: add comment before this line with somethig like that
```
2 PROVINCIA / CITTA' METROPOLITANA	1	SERVIZI PROVINCIALI / CITTA' METROPOLITANA	
Tributi e diritti gestiti direttamente dall'Ente Provinciale.	
138	Canone Unico Patrimoniale - CORPORATE	
TS	(SOLUZIONE CENTRALE GESTITA DA PAGOPA) 
La Legge di Bilancio 2020 (Legge n. 160/2019) ha previsto, a decorrere dal 1° gennaio 2021, l’istituzione  di un “canone unico patrimoniale” 
che unifica in un solo prelievo TOSAP, COSAP, l’imposta comunale sulla PUBBLICITÀ e AFFISIONI, ed altre imposte locali.
Questa soluzione è gestita centralmente da Pagopa S.p.a.	
36	9/0201138TS/	05/03/2021	01/01/2080	Attività produttive e commercio + Altro	
```
to give label `0201138TS` context

File: scripts/update_cup_iban.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    def update_entities(self):
+
+        print("[INFO][update_entities] connecting to ecconfigtable")
+        try:
+            with TableClient.from_connection_string(self.connection_string, table_name=self.table_base) as table:
+
+                print("[INFO][update_entities] iterating over ecconfigtable")
+                entities = list(table.list_entities())
+                for i, entity in enumerate(entities):
```
Comment (pasqualespica) [inline scripts/update_cup_iban.py:38]: this script should be run after init of `pagopapcanoneunicosaecconfigtable` ?
what is the first step before `python3 update_cup_iban.py  ...` 

i.e. this script will update the ibans but which one initially creates the entities?

**Review by pasqualespica — CHANGES_REQUESTED** [review]
LGTM 👍

File: scripts/update_cup_iban.py (hunk 2 above)
Comment (aferracci) [reply scripts/update_cup_iban.py:38]: The first script to launch is `buildEcConfigTable.py` but we must take into account that it requires to be reviewed as it is currently based on data downloaded from PdA that has been dismissed, and at the moment we don't have the complementary service exposed by SelfCare/Area Riservata.
Anyway I'm going to better explicit this situation in readme file

**Review by pasqualespica — APPROVED** [review]
👍

**Pull request merged**